core/src/rpc.rs

```diff
@@ -798,6 +803,10 @@ impl RpcSol for RpcSolImpl {
         let signature = transaction.signatures[0];
         let now = Instant::now();
         let mut signature_status;
+        let signature_timeout = match &commitment {
+            Some(config) if config.commitment == CommitmentLevel::Max => 30,
```

Comment:
You'll need to flop this to be:
```
Some(config) if config.commitment == CommitmentLevel::Recent => 5,
_ => 30
```
If no commitment is provided (`config.commentment = None`) JsonRpcRequestProcessor::get_signature_confirmation_status will default to rooted bank